python/taichi/lang/matrix.py

```diff
@@ -30,59 +30,23 @@ class Matrix(TaichiOperations):
         dt (DataType): the elmement data type.
         shape ( Union[int, tuple of int], optional): the shape of a matrix field.
         offset (Union[int, tuple of int], optional): The coordinate offset of all elements in a field.
-        empty (Bool, deprecated): True if the matrix is empty, False otherwise.
         layout (Layout, optional): The filed layout (Layout.AOS or Layout.SOA).
         needs_grad (Bool, optional): True if used in auto diff, False otherwise.
         keep_raw (Bool, optional): Keep the contents in `n` as is.
-        rows (List, deprecated): construct matrix rows.
-        cols (List, deprecated): construct matrix columns.
     """
     is_taichi_class = True
 
-    # TODO(archibate): move the last two line to **kwargs,
-    # since they're not commonly used as positional args.
     def __init__(self,
                  n=1,
                  m=1,
                  dt=None,
-                 shape=None,
-                 offset=None,
```

Comment:
offset/layout/needs_grad are also deleted but doc string is still there?

Reply:
Oh sorry. My latest commit fixed it.